native/src/java_wrap.cpp

```diff
@@ -665,6 +667,36 @@ jfieldID jfield_NativeTransportStop_pTStopExit_y31s = NULL;
 jfieldID jfield_NativeTransportStop_pTStopExit_refs = NULL;
 jmethodID jmethod_NativeTransportStop_init = NULL;
 
+jclass jclass_ArrayList;
```

Comment:
this method bad method to add points via jni,

whole jni with java_wrap.cpp / java_wrap.h should be simplified as much as possible probably to 2-3 methods